reference/technical_vision/1.5/catalog_bundle.rst

```diff
@@ -0,0 +1,53 @@
+Catalog Bundle & Component [WIP]
+================================
+
+We've extracted following classes and interfaces from the Catalog bundle to the Catalog component:
+ - model interfaces and classes as ProductInterface
+ - repository interfaces as ProductRepositoryInterface
+ - builder interfaces as ProductBuilderInterface
+
+In v1.4, we've re-worked the file storage system, the model `Pim\Component\Catalog\Model\ProductMediaInterface` is not used anymore, we now use `Akeneo\Component\FileStorage\Model\FileInfoInterface`.
+
+In v1.5, we've removed following deprecated classes, interfaces and services:
+ - `Pim\Component\Catalog\Model\ProductMediaInterface`
+ - `Pim\Component\Catalog\Model\AbstractProductMedia`
+ - `Pim\Component\Catalog\Model\ProductMedia`
+ - `Pim\Bundle\CatalogBundle\Factory\MediaFactory` and `@pim_catalog.factory.media`
+ - `Pim\Bundle\CatalogBundle\MongoDB\Normalizer\MediaNormalizer`
+ - `Pim\Bundle\TransformBundle\Normalizer\MongoDB\ProductMediaNormalizer`
+ - `PimEnterprise\Bundle\WorkflowBundle\DependencyInjection\Compiler\RegisterProductValuePresentersPass`
+ - `PimEnterprise\Bundle\WorkflowBundle\Presenter\ProductValue\BooleanPresenter`
+ - `PimEnterprise\Bundle\WorkflowBundle\Presenter\ProductValue\DatePresenter`
+ - `PimEnterprise\Bundle\WorkflowBundle\Presenter\ProductValue\FilePresenter`
+ - `PimEnterprise\Bundle\WorkflowBundle\Presenter\ProductValue\ImagePresenter`
+ - `PimEnterprise\Bundle\WorkflowBundle\Presenter\ProductValue\ProductValuePresenterInterface`
+ - `PimEnterprise\Bundle\WorkflowBundle\Twig\ProductValuePresenterExtension`
+
+We've also removed following requirements from composer.json, you can do the same in your project:
```

Comment:
remove it and don't keep it in the UPGRADE.md. The composer.json has to be copy from the new downloaded version and paste in the old PIM project. So those requirements will be automatically removed.
Or maybe we can add a chapter about dependencies, what we removed, what we changed (also mongodb has been updated and batch bundle removed).